docker-compose.yaml

```diff
@@ -31,14 +31,24 @@ services:
     volumes:
       - ${INSTALLATION_DIR}/${REDACT_LICENSE_FILE}:/license.bal
     environment:
-      REDACT_INFER_METRICS_URL: 'redact-gpu:8000'
-      REDACT_GPU_URL: 'redact-gpu:8001'
-      DISABLE_FACES: ${DISABLE_FACES}
-      DISABLE_LICENSE_PLATES: ${DISABLE_LICENSE_PLATES}
-      DISABLE_BLUR: ${DISABLE_BLUR}
-      DISABLE_DNAT: ${DISABLE_DNAT}
-      DISABLE_MASK: ${DISABLE_MASK}
-      DISABLE_FULL_BODIES: ${DISABLE_FULL_BODIES}
+      - REDACT_INFER_METRICS_URL=redact-gpu:8000
+      - REDACT_GPU_URL=redact-gpu:8001
+      - DISABLE_FACES
```

Comment:
I was looking at [this documentation](https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/#additional-information) which says: 

_You can choose not to set a value and pass the environment variables from your shell straight through to your containers._ ... _Note that in this case no warning is issued if the DEBUG variable in the shell environment is not set._

From this I assumed that variables will be passed if they have a value set in the environment, and will not be set if they are unset in the environment. This is also the results I got when testing, but I guess it's don't really explicitly saying that it will pass the variable as unset.

Reply:
OK, thanks. We'll have to live with that then I think.